evm_loader/program/config/elf_params.toml

```diff
@@ -0,0 +1,15 @@
+[env]
+neon_pkg_version = "CARGO_PKG_VERSION"
+neon_revision = "NEON_REVISION"
+
+[debug_formatted]
+neon_seed_version = "ACCOUNT_SEED_VERSION"
+neon_token_mint_decimals = "token_mint::DECIMALS"
+neon_chain_id = "CHAIN_ID"
+neon_compute_units = "COMPUTE_BUDGET_UNITS"
+neon_heap_frame = "COMPUTE_BUDGET_HEAP_FRAME"
+neon_additional_fee = "REQUEST_UNITS_ADDITIONAL_FEE"
+neon_pool_count = "TREASURY_POOL_COUNT"
+
+[display_formatted]
+neon_pool_seed = "TREASURY_POOL_SEED"
```

Comment:
I suggest formatting all values using the `display` form.
And let's rename config values in NeonEVM code (`ACCOUNT_SEED_VERSION` and so on) instead introduce the file by mapping these names.

Reply:
A lot of the values are formatted as debug, won't format all the values as display break the current code?

Reply:
In the fact all values has type integer or public key. This types has same representation in debug and display form. We also have one value with string type and I changed it to display form, due to problem with ''.
You can check that this changes doesn't break compatibility by comparing output for `neon-cli … neon-elf-params` for old and new version of evm_loader.so